workspaces/orchestrator/plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -798,9 +825,42 @@ function setupInternalRoutes(
             allWorkflowIds,
           );
 
+        const isUserAuthorizedForInstancesAdminView: boolean = // This permission will let user see All instances (including ones others created)
+          await isUserAuthorizedForInstancesAdminViewPermission(
+            req,
+            permissions,
+            httpAuth,
+          );
+
+        const requestFilters = getRequestFilters(req);
+
+        let filters: Filter | undefined = requestFilters;
+
+        if (!isUserAuthorizedForInstancesAdminView) {
+          const EntityFilter: NestedFilterNested = {
+            operator: 'EQ',
+            value: entity,
+            field: 'initiatorEntity',
+          };
+          const nestedEntityFilter: NestedFilter = {
```

Comment:
the names are confusing. i would use this name for the one above and call this one variablesFilter